modules/core/src/main/java/org/apache/ignite/internal/client/thin/ProtocolBitmaskFeature.java

```diff
@@ -37,6 +37,9 @@ public enum ProtocolBitmaskFeature {
      */
     CLUSTER_API(2),
 
+    /** Client discovery. */
+    CLUSTER_GROUP_GET_NODES_ENDPOINTS(3),
```

Comment:
See `ClientOperation.java`
```suggestion
    CLUSTER_GROUP_GET_NODE_ENDPOINTS(3),
```

Reply:
`NODES_ENDPOINTS` is the correct name, because we returns a list of nodes with multiple endpoints per node.

Reply:
Let's keep it the same way as in Ignite to make merges easier.